package.json

```diff
@@ -27,9 +27,9 @@
     "start": "node server.js",
     "test": "env-cmd ./test.env mocha --exit tests/*.test.js",
     "env": "env",
-    "postinstall": "webpack --mode production --config webpack.config.js && npm run-script update-buster",
-    "bundle": "webpack --mode production --config webpack.config.js && npm run-script update-buster",
-    "bundle-dev": "webpack --mode development --config webpack.config.js && npm run-script update-buster",
+    "postinstall": "webpack --mode production --config webpack.config.js && webpack --mode production --config webpack-clock.config.js && npm run-script update-buster",
```

Comment:
instead of doing webpack 2 times with 2 configs we could probably add an extra `entry` to the main webpack

something like this:

```
entry: {
    app: './bundle/bundle.source.js'
    clocks: './bundle/bundle.clocks.source.js'
}
...
output: {
    ...
    filename: 'js/bundle.[name].js'
}
```